src/Files.App/ViewModels/ItemViewModel.cs

```diff
@@ -1973,6 +1972,11 @@ async Task HandleChangesOccurredAsync()
 					{
 						operationEvent.Reset();
 
+						// Keep track of renaming information available
+						int RenameIndex = -1;
+						string OldName = "";
+						string NewName = "";
```

Comment:
```suggestion
						int renameIndex = -1;
						string oldName = string.Empty;
						string newName = string.Empty;
```

You should use lowercase names for variable declarations. I think that using `string.Empty` will make the code more readable than `""`